website/docs/docs/build/incremental-overview.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "About Incremental models"
+description: "Learn about incremental models, when to use them, and how they work in dbt."
```

Comment:
Great idea breaking this page up into it's own section! For our search and Google results, it might be beneficial to leave the word "Models" in the title and ID.

The descriptions for this page and the existing are very similar. To further distinguish between the two, it could help to have the word "Introduction" in this title and "Configuring" in the older article. So it would be
Introduction to incremental models 
and
Configuring incremental models 

to really drive home the differences between these two pages when folks are looking.

Reply:
i did try to be more explicit but what i didn't like was that the 'configure' was adding a 2nd line so i opted out. but it might be best to make sure it's super clear